frontend/src/components/funding/FundingAgreementsTab.vue

```diff
@@ -149,7 +149,8 @@ export default {
       }
     },
     async loadIrregularExpenses(activeFA) {
-      const expenseApplications = await IrregularExpenseService.getIrregularExpenseApplications(activeFA.applicationId, IRREGULAR_EXPENSE_STATUS_CODES.APPROVED)
+      const getPCMValidation = true
```

Comment:
I don't think this constant adds much value as it's just passed into the service call.

Reply:
I can take it out - I just thought it might be less confusing to look at instead of seeing just a random bool!

Reply:
If anything the service should document it :).